src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -85,6 +86,42 @@ public PersistentDataStore() {
     return users;
   }
 
+  /**
+   * Loads all User objects who are admins from the Datastore service and returns them in a List.
+   *
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *     Datastore service
+   */
+  public List<User> loadAdmins() throws PersistentDataStoreException {
+
+    List<User> admins = new ArrayList<>();
+
+    // Retrieve all users from the datastore.
+    Query query = new Query("chat-users");
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      try {
+        UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
+        String userName = (String) entity.getProperty("username");
+        String password = (String) entity.getProperty("password");
+        Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
+        String about = (String) entity.getProperty("about");
+        Boolean isAdmin = (Boolean) entity.getProperty("isAdmin");
+        User admin = new User(uuid, userName, password, about, creationTime, isAdmin);
+        if (isAdmin) {
+          admins.add(admin);
```

Comment:
This should go at the beginning to save from executing code that doesn't need to be completed. I.e.:

```
boolean isAdmin = ...
if (!isAdmin) {
    continue;
}

String userName = ...
...
admins.add(admin);
```